packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/design.md

```diff
@@ -0,0 +1,133 @@
+* **Compact styling**, when you want to maximize the amount of data that can be displayed in a small space.
+
+### Standard Data Table
+
+![standard data table](img/standard-data-table.png)
+
+1. **Toolbar:** The toolbar sits above the list and contains controls for manipulating table data. Common actions include filtering, sorting, and pagination.
+2. **Select all:** When present, selects all items in a table. If pagination is being used, this will only select items on the current page.
+3. **Column headers:** If the user is able to sort on a column, the first click on the header will sort the content of the table on the content in that column. Subsequent clicks will toggle the direction of the sort. Table data can only be sorted on one column at a time.
+4. **Select checkbox:** Select this row.
+5. **Global actions:** These actions apply to all selected items.
+6. **Inline actions:** These actions apply only to the current row/item.
+7. **Pagination footer:** If present, navigation to a new page.
+
+#### When to use
+Use a Data Table when the information you want to display is fits into a structured, tabular format, i.e. has distinct rows and columns.
+
+#### When not to use
+The Data Table is not recommended for less structured of variably structured data that cannot be easily organized into columns. In these situations a Data Table should be used.
+
+#### How to use
+Consider the structure of the data you want to display and organize that information into columns. Columns will typically have column headers. Every row within a table must have a consistent format, and if the table row includes actions, they should always be placed in the rightmost column(s).
+
+### Compact Data Table
+
+![compact data table](img/compact-data-table.png)
+
+When to use
+Use a Compact Data Table when you want to show as much data per page as possible and readability is a secondary concern.
+
+
+### Expandable Data Table
+
+![expandable data table](img/expandable-data-table.png)
+
+1. **Expansion:** When present, expands the row.
+2. **Expansion panel:** contains details associated with a row.
+
+When to use
+Use when you have more information than will comfortably fit inside a row or you want to provide a way for advanced users to access information that is not applicable to all users.
+
+### Compound-Expandable Data Table
+
+![compound expandable data table](img/compound-expandable-data-table.png)
+
+1. **Expandable cell:** The user can click on any expandable cell to reveal more detail about an item. If the expansion for an item is already open, clicking on a different cell will close the current item and open a new one.
+2. **Expansion panel:** contains details associated with an expandable item.
+
+#### When to use
+Use a compound expandable list when you want multiple expansion panels that relate to specific table columns where it would not make sense to combine all of this information into a single, simple expansion.
+
+## Components and demos used
```

Comment:
I am actually digging having this content at the bottom of the guideline. Maybe we should move this globally. LMK what you think!